var/spack/repos/builtin/packages/lua/package.py

```diff
@@ -58,24 +58,34 @@ class Lua(Package):
         destination="luarocks",
         placement='luarocks')
 
+    patch('liblua-shared.patch')
```

Comment:
Add a comment with a link to Arch linux!

Reply:
@junghans see the last comment https://github.com/LLNL/spack/pull/6008#issuecomment-340100568